Tentacle/RepositoryInfo.swift

```diff
@@ -75,25 +76,27 @@ public func ==(lhs: RepositoryInfo, rhs: RepositoryInfo) -> Bool {
 }
 
 extension RepositoryInfo: ResourceType {
-    public static func decode(j: JSON) -> Decoded<RepositoryInfo> {
+    public static func decode(_ j: JSON) -> Decoded<RepositoryInfo> {
         let f = curry(RepositoryInfo.init)
 
-        return f
+        let ff = f
```

Comment:
Just to understand what's happening, you're splitting the mapping so it's less complicated for Swift?